connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorCheckpointTask.java

```diff
@@ -190,4 +208,63 @@ public void commitRecord(SourceRecord record) {
             Checkpoint.unwrapGroup(record.sourcePartition()),
             System.currentTimeMillis() - record.timestamp());
     }
+
+    private void syncGroupOffset() throws InterruptedException {
+
+        for (String group : consumerGroups) {
+            try {
+                List<Checkpoint> checkpoints = listConsumerGroupOffsets(group).entrySet().stream()
+                    .filter(x -> shouldCheckpointTopic(x.getKey().topic()))
+                    .map(x -> checkpoint(group, x.getKey(), x.getValue()))
+                    .filter(x -> x.downstreamOffset() > 0)  // ignore offsets we cannot translate accurately
+                    .collect(Collectors.toList());
+
+                DescribeConsumerGroupsResult describedGroups = targetAdminClient.describeConsumerGroups(
```

Comment:
It might be slightly more efficient to describe all consumerGroups once (outside the for-loop) and then use the resulting map here.